web/gui/dashboard_info.js

```diff
@@ -586,29 +586,35 @@ netdataDashboard.menu = {
     },
 
     'mssql': {
-        title: 'MS SQL Server',
+        title: 'SQL Server',
         icon: '<i class="fas fa-database"></i>',
         info: undefined
     },
 
     'ad': {
-        title: 'AD Domain Service',
+        title: 'Active Directory',
         icon: '<i class="fab fa-windows"></i>',
         info: undefined
     },
 
     'adcs': {
-        title: 'AD Certification Service',
+        title: 'Certification Service',
```

Comment:
Yes, I observed that `AD Directory Service` became `Active Directory`. If I removed only in the first we would have again a break of pattern. This was the motive I removed `AD` from others.

@shyamvalsan, what do you think about new pattern? Should we remove `AD` from `cloud` too, or are we going to keep `AD` for Certification Service and Federation Service.

Reply:
Where do you see this new pattern? Please check the provided link, it has "AD Certification Service".

Reply:
This is the broken pattern I am trying to avoid. 
I plan to make a new PR "fixing" cloud too. Why should we discard  `AD` only for one instead all three? (ping @shyamvalsan )